app-modules/engagement/src/Policies/EngagementPolicy.php

```diff
@@ -140,4 +144,9 @@ public function forceDelete(Authenticatable $authenticatable, Engagement $engage
             denyResponse: 'You do not have permission to permanently delete this engagement.'
         );
     }
+
+    public function dead(Authenticatable $authenticatable): Response
+    {
+        return Response::deny('dead');
+    }
```

Comment:
What is this for?

Reply:
> What is this for?

Oops ! I guess, I left my superior debugging skills here. Never mind I removed it 😛